src/Facility.Definition/ServiceTypeKind.cs

```diff
@@ -60,6 +60,16 @@ public enum ServiceTypeKind
 	/// </summary>
 	Enum,
 
+	/// <summary>
+	/// An external DTO.
+	/// </summary>
+	ExternalDto,
+
+	/// <summary>
+	/// An external enumerated type.
+	/// </summary>
+	ExternalEnum,
```

Comment:
Changing the order of enum values is a binary breaking change. These should be added to the end.

Reply:
Thanks, should be fixed in 0b4e3f0b